var/spack/repos/builtin/packages/glib/package.py

```diff
@@ -30,11 +30,14 @@ class Glib(Package):
        providing data structure handling for C, portability wrappers
        and interfaces for such runtime functionality as an event loop,
        threads, dynamic loading and an object system."""
+
     homepage = "https://developer.gnome.org/glib/"
     url      = "http://ftp.gnome.org/pub/gnome/sources/glib/2.42/glib-2.42.1.tar.xz"
 
     version('2.49.4', 'e2c87c03017b0cd02c4c73274b92b148')
     version('2.42.1', '89c4119e50e767d3532158605ee9121a')
+    version('2.48.1', '67bd3b75c9f6d5587b457dc01cdcd5bb',
+            url='http://ftp.gnome.org/pub/GNOME/sources/glib/2.48/glib-2.48.1.tar.xz')
```

Comment:
You shouldn't need to specify the url for this one. Also, can you make sure the versions are in order from newest to oldest?